auslib/admin/views/releases.py

```diff
@@ -241,8 +241,12 @@ def _put(self, release, changed_by, transaction):
                 msg = "Couldn't update release: %s" % e
                 cef_event("Bad input", CEF_WARN, errors=msg)
                 return Response(status=400, response=json.dumps({"data": e.args}))
+            except OutdatedDataError as e:
+                msg = "Couldn't update release: %s" % e
+                cef_event("Bad input", CEF_WARN, errors=msg)
+                return Response(status=400, response=json.dumps({"data": e.args}))
             data_version += 1
-            return Response(json.dumps(dict(new_data_version=data_version)), status=200)
+            return Response(json.dumps(dict(new_data_version=data_version)), status=200)            
```

Comment:
You've got the general idea here, but I think we can deal with this more globally if you do this in auslib/admin/views/base.py rather than in each individual view. Can you give that a shot?